jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -68,10 +68,10 @@ blockchain_configuration:
     per_certificate_fees:
       # (optional) if not specified, the pool registration certificate fee will be
       # the one set by linear_fees.certificate
-      certificate_pool_registration: 5
+      certificate_pool_registration: 150000000
       # (optional) if not specified, the delegation certificate fee will be
       # the one set by linear_fees.certificate
-      certificate_stake_delegation: 2
+      certificate_stake_delegation: 400000
```

Comment:
I don't understand why you needed to change this value?

Reply:
We discussed this with Vincent.  He felt that 2 lovelace was too low.  This is the standard cost for embedding a certificate.

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.